intermediate-python-debugging/problem-set-using-the-debugger.md

```diff
@@ -130,6 +123,7 @@ If you don't see status icons to left of the test names in your test file, you m
 3. Select `tests` as the directory containing the tests.  
 ![Prompt to pick a test directory, with tests selected](../assets/intermediate-python-debugging_problem-set-using-the-debugger_pick-directory.png)
 
+If you do not see a popup prompting you to "Enable and configure a Test Framework", be sure that you have already set up Pytest for the VS Code debugger. You can review the steps to do so in the previous lesson called "Debugging Tests with Pytest".
```

Comment:
Should we just cut this whole details section, and move the suggestion to refer to `Debugging Tests with Pytest` to be in the callout? I was going to make a few suggestions on additional detail here, but since that would be mostly duplicating the previous content, can we fully prune the duplication out?